test/404s.test.js

```diff
@@ -817,7 +834,7 @@ test('recognizes errors from the http-errors module', t => {
   const fastify = Fastify()
 
   fastify.get('/', function (req, reply) {
-    reply.send(httpErrors.NotFound())
+    reply.send(httpErrors.Forbidden())
```

Comment:
Why this change?

Reply:
Now that I've read it better, I misunderstood the test.

Reply:
updated, PTAL